### ISSUE TITLE
Change the order of how medals are shown on the profile

### ISSUE THREAD

**Issue by Exploity** [ppy/osu-web#8628]
As someone with a high amount of medals, I've noticed that it would make more sense, be cleaner and space efficient to just simply flip around the positions of the Hush-Hush and Dedication sections.

<p align="center">
  <img width="400" src="https://user-images.githubusercontent.com/40037713/153968985-59ef0bf3-682c-4cef-8552-9dd8165a5053.png">
</p>

**Comment by peppy** [comment]
@arflyte is the medals section part of the work you're currently doing on the user profile page? if so, want to take this feedback into account?

**Comment by arflyte** [comment]
There's a major change to the medals section that will be more space-efficient.

**Comment by ayemme** [comment]
How will this affect the phone version of the website? Will the order of the medels change, because in my phine, all the badges are displayed in a single row

**Comment by ayemme** [comment]
![Uploading Screenshot_20220228-185441.png…]()

**Comment by RaFaReAcH** [comment]
Is there a reason not to implement this in the meantime, even if a major overhaul is in the works? Is it coded in a way that is hard to move? At least, this change would not be controversial.

**Comment by peppy** [comment]
I'll take a look into this, may be able to easily change the order from my end.

**Comment by nanaya** [comment]
I'm thinking a single column may be more efficient in term of space since we wouldn't need to balance between groups.

And I believe the column pairings may be different as groups with no medal will be hidden.

**Comment by peppy** [comment]
Yeah that's probably fine. I think the best immediate change is moving both the categories with a huge number of medals to the bottom

**Comment by peppy** [comment]
Could someone please link a profile which is affected by this? I'm having trouble finding a good example.

**Comment by agatemosu** [comment]
[peepee](https://osu.ppy.sh/users/10520757)

**Comment by agatemosu** [comment]
https://osu.ppy.sh/users/4687701

**Comment by peppy** [comment]
It looks like the sections are currently ordered alphabetically and can't be changed via database:

![CleanShot 2023-10-31 at 03 27 21](https://github.com/ppy/osu-web/assets/191335/53a636d1-6b1d-49c7-ae47-914ac0c57c19)

so not much I can do here without adding a new column for grouping-ordering ☹️.
